GeometryCore/include/Mesh.h

```diff
@@ -8,17 +9,29 @@ namespace geom_utils
 {
     class Mesh
     {
-        std::vector<FPoint3D> vertices;
+        std::vector<Triangle3D> vertices;
```

Comment:
This is not vertices now. More appropriate would be triangles or facets.

Reply:
fixed

Reply:
Have you pushed your fixes? coz i don't see

Reply:
it is not pushed yet, because i have not fixed the trouble with absolute path. All changes are local